pkgs/development/python-modules/typish/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "typish";
+  version = "1.9.3";
+  format = "wheel";
```

Comment:
wheels are discouraged as they likely lack tests, if possible, please try to package this using source and ensure that tests are being ran

Reply:
Switched it to pull from source and run tests.  One of the tests unfortunately depends on a super old version of `nptyping` which appears to circular depend back on `typish`.  All the other tests pass.